.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up MySQL
+      uses: mirromutth/mysql-action@v1.1
```

Comment:
do we know which MySQL version this action uses?

Reply:
Latest by default. Is there a version you'd prefer?